cookbook/deployment/workflow/lp_schedules.py

```diff
@@ -67,15 +77,97 @@ def positive_wf(name: str):
     print(f"{reminder}")
 
 
-fixed_rate_lp = LaunchPlan.create(
-    "my_fixed_rate_lp",
-    positive_wf,
+fixed_rate_lp = LaunchPlan.get_or_create(
+    name="my_fixed_rate_lp",
+    workflow=positive_wf,
     # Note that the workflow above doesn't accept any kickoff time arguments.
     # We just omit the ``kickoff_time_input_arg`` from the FixedRate schedule invocation
     schedule=FixedRate(duration=timedelta(minutes=10)),
     fixed_inputs={"name": "you"},
 )
 
+This fixed-rate scheduler runs every ten minutes. Similar to a cron scheduler, a fixed-rate scheduler also accepts ``kickoff_time_input_arg`` (which is omitted in this example).
 # %%
+# Activating a Schedule 
+# #####################
+
 # Once you've initialized your launch plan, don't forget to set it to active so that the schedule is run.
-# TBD (katrogan)
+# You can use pyflyte in container:
+pyflyte lp -p {{ your project }} -d {{ your domain }} activate-all
+
+# %%
+# Or with flyte-cli view and activate launch plans:
+flyte-cli -i -h localhost:30081 -p flyteexamples -d development list-launch-plan-versions
+ 
+# %%
+# Extract the URN returned for the launch plan you're interested in and make the call to activate it:
+flyte-cli update-launch-plan -i -h localhost:30081 --state active -u {{ urn }}
+
+# %%
+# Verify your active launch plans:
+flyte-cli -i -h localhost:30081 -p flyteexamples -d development list-active-launch-plans
+
+# %%        
+# Platform Configuration Changes
+# ##############################
+# 
+# Scheduling features requires additional infrastructure to run so these will have to be created and configured.
+# 
+# Setting up Scheduled Workflows
+# ------------------------------
+# In order to run workflow executions based on user-specified schedules you'll need to fill out the top-level ``scheduler`` portion of the flyteadmin application configuration.
```

Comment:
once the helm changes are checked in, it would be helpful to link to that section of the values.yaml here